src/compas_viewer/scene/meshobject.py

```diff
@@ -18,19 +23,23 @@ class MeshObject(ViewerSceneObject, BaseMeshObject):
     ----------
     mesh : :class:`compas.datastructures.Mesh`
         A COMPAS mesh.
+    vertexcolor : Union[Dict[Any, :class:`compas.colors.Color`], :class:`compas.colors.Color`]], optional
+        The vertex color. Global settings in the viewer will be used if not specified.
```

Comment:
i would rephrase this everywhere to "Default is the value of `pointcolor` in `viewer.config`."